cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/ServiceImpl.kt

```diff
@@ -468,7 +468,7 @@ private class ServiceImpl(
 
         networkClient.storeCcpaChoice(postConsentParams)
             .executeOnRight { postConsentResponse ->
-                campaignManager.ccpaConsentStatus?.uuid = postConsentResponse.uuid
+                campaignManager.ccpaUuid = postConsentResponse.uuid
```

Comment:
@andresilveirah since right now we still are dependant on the `ccpaUuid` from `campaignManager`, we have to update that value.

And since in the next line we update `ccpaConsentStatus` anyways, we don't need to update `campaignManager.ccpaConsentStatus?.uuid` directly.

@carmelo-iriti created a ticket to refactor the code to remove `ccpaUuid` and `gdprUuid`, so we will do that in the future, but not in this PR.